module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -849,6 +874,22 @@ public function getHolding($bibId, array $patron = null, array $options = [])
                 }
                 $nextBatch[] = $nextItem;
             }
+
+            // If there are no item records at this location, we're going to create a fake one,
+            // fill it with data from the FOLIO holdings record, and make it not appear in
+            // the full record display using a non-visible AvailabilityStatus.
+            if ($number == 0) {
+                $stubVuFindItem = $this->makeStubVuFindItem($holding->effectiveLocationId, $holdingDetails);
+                $invisibleAvailabilityStatus = new AvailabilityStatus(true, 'See full record');
```

Comment:
What led to the choice of the values here? `See full record` does not seem to be a string currently included in our language files. Does this actually end up being displayed somewhere? If so, we should make sure the i18n details are in order (either revise to use an existing string, or add a new string to the en.ini file so it can be translated). If it's not displayed anywhere, then maybe we should use different initial values to avoid confusion.

Reply:
Yes, this is displayed on the search results screen. How do I queue up a string for translation? (Do I just add it to the en.ini file and magic will happen elsewhere?)
![image](https://github.com/user-attachments/assets/87483e65-d494-4a67-b572-8609c8025442)

Reply:
Yes, all you need to do is add a line to en.ini. I'd suggest creating a descriptive key instead of using the raw text as the key -- e.g. something like `virtual_item_availability_status_message = "See full record"` so it can be customized more easily and so the translators have a little more context.